omod/src/main/java/org/openmrs/module/mchapp/fragment/controller/AntenatalTriageFragmentController.java

```diff
@@ -33,14 +35,49 @@
 
     private int visitTypeId;
 
-    public void controller(FragmentModel model, FragmentConfiguration config, UiUtils ui) {
+    public void controller(FragmentModel model, FragmentConfiguration config, UiUtils ui, HttpServletRequest request) {
         config.require("patientId");
         config.require("queueId");
+        String enc = request.getParameter("encounterId");
+        
+        if(StringUtils.isNotEmpty(enc)){
```

Comment:
@kimani-njoroge move this to a method that prepares the model (i.e. adds vital stats details)